app/[locale]/account/connections/__tests__/page.spec.tsx

```diff
@@ -1,11 +1,22 @@
-import React from 'react';
+import React, { ReactNode } from 'react';
 import { render, screen, act, waitFor } from '@testing-library/react';
 import { axe, toHaveNoViolations } from 'jest-axe';
 
 expect.extend(toHaveNoViolations);
 
 import ConnectionsPage from '../page';
 
+jest.mock('@/components/PageHeader', () => {
```

Comment:
Updated unit to accommodate the use of the new, shared PageHeader component in the page.